pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -279,6 +376,17 @@ private void addQueryQuotaCapacityUtilizationRateTableGaugeIfNeeded(String table
     }
   }
 
+  @Override
+  public boolean acquireDatabase(String databaseName) {
+    // Return true if query quota is disabled in the current broker.
+    if (isQueryRateLimitDisabled()) {
+      return true;
+    }
+    LOGGER.info("Trying to acquire token for database: {}", databaseName);
```

Comment:
this can be very verbose, as it's triggered for each query?

Reply:
Yes that's true. Will change the logic to skip the log when `queryQuota == null` so that its printed only for database queries that have quotas enabled on the database.
Need to make similar updates for the table acquire part as well.
Will also lower its level to debug as `tryAcquireToken` prints the log when acquire fails.
Thanks for pointing it out.